revamp-service/src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@ spring.cache.ehcache.config=classpath:ehcache.xml
 #spring.datasource.url=jdbc:h2:mem:puthuyir;MODE=PostgreSQL;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 spring.datasource.url=jdbc:mysql://localhost:3306/revamp_db?allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC
 spring.datasource.username=root
-spring.datasource.password=putholi
```

Comment:
Is it possible to change the password as putholi and check-in again? I think everyone is using the same password.